### ISSUE TITLE
Plugin Header fix

### ISSUE THREAD

**Issue by erikyo** [wp-blocks/make-pot#9]
As @johnhooks noticed some headers in the pot file are missing and the placeholder is shown. 

```
  # Copyright (C) 2024 pluginName
  # This file is distributed under the GPL-2.0-only license.
  msgid ""
  msgstr ""
  "Project-Id-Version: PLUGIN NAME 2.0.3\n"
  "Report-Msgid-Bugs-To: AUTHOR EMAIL **[object Object]**\n"
  "MIME-Version: 1.0\n"
  "Content-Transfer-Encoding: 8bit\n"
  "Content-Type: text/plain; charset=iso-8859-1\n"
  "Plural-Forms: nplurals=2; plural=(n!=1);\n"
  "POT-Creation-Date: 2024-03-16T01:41:57.862Z\n"
  "PO-Revision-Date: 2024-MO-DA HO:MI+ZONE\n"
  "Last-Translator: me <EMAIL>\n"
  "Language-Team: me <EMAIL>\n"
  "X-Generator: @wp-blocks/make-pot 1.0.0\n"
  "Language: LANGUAGE\n"
  "X-Domain: \n"
```

This is due to the fact that the metadata gathered are not correctly merged with the header data, we have to merge all collected types using a common pattern (eg. package json use the term "name" for the plugin name etc)

**Comment by johnhooks** [comment]
@erikyo agreed, I noticed the same thing, I'll pull down PR #10 and see if it fixes it for the project I am testing on.

**Comment by erikyo** [comment]
closed with #10
